boilerplate/app/components/Screen.tsx

```diff
@@ -159,12 +160,17 @@ function ScreenWithScrolling(props: ScreenProps) {
     style,
   } = props as ScrollScreenProps
 
+  const ref = useRef<ScrollView>()
+
   const { scrollEnabled, onContentSizeChange, onLayout } = useAutoPreset(props as AutoScreenProps)
 
+  // Add native behavior of pressing the active tab to scroll to the top of the content
+  // More info at: https://reactnavigation.org/docs/use-scroll-to-top/
+  useScrollToTop(ref)
```

Comment:
Do you know, is this a no-op if the screen is inside a stack or a nav other than tab?

Reply:
@yulolimum `react-navigation` early returns from the `useEffect` if there is no tab nav found, so no listener would be set up in that case.

<img width="638" alt="image" src="https://user-images.githubusercontent.com/374022/186232549-2bbf2ad2-b4e0-4ca0-878b-c6567f2dee11.png">

See this file for more info:
https://github.com/react-navigation/react-navigation/blob/main/packages/native/src/useScrollToTop.tsx#L56